### DISCUSSION TITLE
Want to Work on NavigationActivation

### DISCUSSION THREAD

**Discussion by jpmedley** [mdn/mdn-community#675] [Content]
Gang,

I need to refresh my knowledge of this arena and get caught up on what I've missed. I propose to document [`NavigationActivation`](https://chromestatus.com/feature/5076557983121408), which landed in Chrome 123. Am I getting into anyone's swim lane?

Joe

**Comment by teoli2003** [comment]
Welcome back.

Pinging @chrisdavidmills who is likely the only person who may swim near this.

**Comment by chrisdavidmills** [comment]
Hi @jpmedley!

I'm afraid I've already done this as part of https://github.com/mdn/content/pull/32723.

But I can help you to find something else. Is it mainly API features you would be interested in?
